proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -84,6 +84,7 @@ message Proposal {
       [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"voting_start_time\""];
   google.protobuf.Timestamp voting_end_time = 9
       [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"voting_end_time\""];
+  repeated google.protobuf.Any messages = 10;
```

Comment:
I think we've come to the conclusion that we can't safely add new fields to existing messages because it's confusing for clients.

Keplr for instance will need to support clients with the old and new gov module.

But maybe it's okay in this case because people only do this from the CLI?

The safer thing to do here would probably be to either create a new Msg `MsgSubmitProposal2` and deprecate the old one or bump the proto package version.

@amaurym any thoughts?

Reply:
Yeah, just to be safe, let's not add any fields to anything that goes into a Msg, see https://github.com/cosmos/cosmos-sdk/pull/9613 for details.

Probably if we're happy with this new MsgSubmitProposal, I would bump to v1. And in this case afaiu, we can also remove `google.protobuf.Any content = 2` from the Proposal-related proto messages.

Reply:
> And in this case afaiu, we can also remove google.protobuf.Any content = 2 from the Proposal-related proto messages?

The idea was to deprecate it first.